src/llmperf/ray_clients/furiosa_client.py

```diff
@@ -5,11 +5,13 @@
 from llmperf.ray_clients.openai_chat_completions_client import (
     OpenAIChatCompletionsClient,
 )
-from llmperf.ray_llm_client import LLMClient
+from llmperf.ray_clients import LLMClient
 import ray
 from ray.runtime_env import RuntimeEnv
 
 
+FURIOSA_AI_DEFAULT_SYSTEM_PROMPT = "You are a helpful assistant."
```

Comment:
```suggestion
DEFAULT_SYSTEM_PROMPT = "You are a helpful assistant."
```
Furiosa만의 특별한 prompt는 아닌 것 같아, 단순히 `DEFAULT_SYSTEM_PROMPT` 라고만 표기해도 충분하지 않을까 싶습니다.